test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -803,6 +886,23 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
         $response.Content.UserAgent | Should Match $Pattern
     }
 
+    It "Verifies Invoke-WebRequest Certificate Authentication Fails without -Certificate"{
+        $command = 'Invoke-WebRequest https://prod.idrix.eu/secure/'
```

Comment:
@iSazonov awesome. that would be better. It will probably also need  some way to ensure the client authentication cert is received by the listener too or to enforce certificate authentication on the listener so it can fail if one is not provided.

@dantraMSFT  I'm not entirely sure `WebRequest.RegisterPrefix` will work on Core. Unless I'm missing something, I don't think it would affect anything being done with the new classes under `System.Net.Http` in use in Core. I was able to [make a mockup](https://gist.github.com/markekraus/624c7a4969c013e66c357a2d8adeeafc) with it, but it only works on Windows PowerShell (tested on 5.1). Unless I'm going totally in the wrong direction?